apps/dotcom/sync-worker/src/types.ts

```diff
@@ -153,3 +156,9 @@ export type TLUserDurableObjectEvent =
 	  }
 	| { type: 'reboot_duration'; id: string; duration: number }
 	| { type: 'cold_start_time'; id: string; duration: number }
+
+export interface QueueMessage {
+	type: 'asset-upload'
```

Comment:
Went with a more generic approach, which should allow us to process different kinds of messages here.

Alternately we could create a separate queue for each use case.